src/components/Order/Order.tsx

```diff
@@ -95,6 +102,7 @@ class Order extends React.Component<OrderProps, OrderState> {
     this.setState({
       percents: percentage
     });
+    this.props.setMarketTotal(undefined, side);
```

Comment:
undefined as parameter is not a very good practice, null is better (if you can't remove this empty value)